deployment/helm/skaha/skaha-config/launch-contributed.yaml

```diff
@@ -31,6 +31,13 @@ spec:
         supplementalGroups: [${skaha.supgroups}]
       priorityClassName: uber-user-preempt-medium
       hostname: "${software.hostname}"
+      initContainers:
+      - name: setup-access-token
+        image: busybox
+        command: ["sh", "-c", "echo '${access_token}' > /etc/tokens/access-token"]
```

Comment:
Could you make this variable match the others in terms of convention?  `${skaha.accesstoken}` perhaps?

Reply:
I have updated the keys as per the convension.